db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
```

Comment:
- [ ] Based on project requirements, you're expected to add `5` messages in the seed file, which would be the messages your view will render after action by the message controller

### Instruction:
![image](https://github.com/katfogy/hello-rails-react/assets/50990925/25305d72-e326-4b1a-82d8-ab6b5dc36813)